Monika After Story/game/updates.rpy

```diff
@@ -381,9 +381,18 @@ label v0_12_8_1(version="v0_12_8_1"):
         # only overwrite if not set.
         if persistent._mas_nye_accomplished_resolutions is None:
             persistent._mas_nye_accomplished_resolutions = persistent._mas_pm_accomplished_resolutions
+            store.mas_history._store_all(
+                mas_HistLookup_all("pm.actions.did_new_years_resolutions"),
+                "nye.actions.did_new_years_resolutions"
+            )
             safeDel("_mas_pm_accomplished_resolutions")
+
         if persistent._mas_nye_has_new_years_res is None:
             persistent._mas_nye_has_new_years_res = persistent._mas_pm_has_new_years_res
+            store.mas_history._store_all(
+                mas_HistLookup_all("pm.actions.made_new_years_resolutions"),
+                "nye.actions.made_new_years_resolutions"
+            )
```

Comment:
`mas_HistLookup_all` returns data of format `{year: (code, value)}`,
but `_store_all` uses `_store` which accepts `value`, `key`, `year`. This code will save a tuple `(code, value)` as `value`.

Reply:
Something like `{k: v[1] for k, v in mas_HistLookup_all(KEY).items()}` would work I think, unless you know a better way.

Reply:
`_store_all` says its supposed to work with the data tuple from `mas_HistLookup` so I changed `_store_all` instead.